src/Components.js

```diff
@@ -24,6 +24,11 @@ const cellBgColorOf = props => {
   const hoverSelectionBackgroundColor = props.hoverSelectionBackgroundColor || '#333'
   const hoverBackgroundColor = props.hoverBackgroundColor || '#ddd'
   const selectionBackgroundColor = props.selectionBackgroundColor || '#666'
+  const validationErrorBackgroundColor = props.errorBackgroundColor || '#ff726f'
+
+  if (props.invalid) {
+    return validationErrorBackgroundColor
```

Comment:
do we need a `selectedValidationErrorBackgrround` for when the cell is invalid? this is just a questions

Reply:
yeah, so user can know that, particular cell is invalid by its color, and hover to see the message

Reply:
whati mean is that do we need another color for when the invalidCell is selected by the user.